modulus/datapipes/climate/climate_hdf5.py

```diff
@@ -0,0 +1,510 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import h5py
+import numpy as np
+import torch
+from datetime import datetime, timedelta
+import netCDF4 as nc
+
+try:
+    import nvidia.dali as dali
+    import nvidia.dali.plugin.pytorch as dali_pth
+except ImportError:
+    raise ImportError(
+        "DALI dataset requires NVIDIA DALI package to be installed. "
+        + "The package can be installed at:\n"
+        + "https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html"
+    )
+
+from dataclasses import dataclass
+from typing import Iterable, List, Union, Tuple
+from pathlib import Path
+from torch.utils.data import Dataset
+
+from modulus.datapipes.datapipe import Datapipe
+from modulus.datapipes.meta import DatapipeMetaData
+
+from zenith_angle import cos_zenith_angle
```

Comment:
`zenith_angle` currently lives in the `examples` folder and should be moved.

Reply:
Fixed this.